akka-projection-testkit/src/main/scala/akka/projection/testkit/TestProjection.scala

```diff
@@ -182,9 +255,15 @@ class TestProjection[Offset, Envelope] private[projection] (
   }
 }
 
-// FIXME: this should be replaced as part of #198
 @ApiMayChange
 object TestSourceProvider {
```

Comment:
separate TestSourceProvider.scala file

Reply:
Deferring to #381 to avoid conflicts.  I've added it to the TODO.